day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,46 @@
+
+var words = ["Ten", "Nine", "Eight", "Seven", "Six",
+             "Five", "Four", "Three", "Two", "One"];
+
+var wordsLength = words.length;
+
+function speckledFrogs(frogs) {
+  for (i = frogs; i >= 10 ; i--) {
+    if (i > 11 ) {
+      console.log(i + " speckled frogs sat on a log\n" +
+        "eating some most delicious bugs.\n" +
+        "One jumped in the pool where it's nice and cool,\n" +
+        "then there were " + (i - 1) + " speckled frogs.\n");
+  } else if (i === 11) {
+      console.log(i + " speckled frogs sat on a log\n" +
+        "eating some most delicious bugs.\n" +
+        "One jumped in the pool where it's nice and cool,\n" +
+        "then there were ten speckled frogs.\n");
+  } else if (i < 11) {
+      speckledFrogsWords();
+    }
+  };
+};
+
+function speckledFrogsWords(frogs) {
```

Comment:
Good idea to move more specialized code out into its own function. On this line the `frogs` of `function speckledFrogsWords(frogs)` is not used in the function, you could declare `function speckledFrogsWords()` without the given argument.